<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Items</class>
 <widget class="QWidget" name="Items">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Items</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTableWidget" name="itemsTable">
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustToContents</enum>
         </property>
         <property name="columnCount">
          <number>10</number>
         </property>
         <column/>
         <column/>
         <column/>
         <column/>
         <column/>
         <column/>
         <column/>
         <column/>
         <column/>
         <column/>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="projectsBtn">
         <property name="text">
          <string>Back to Projects</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="loggerLayout"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
